src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -24,6 +26,22 @@ class Task:
     notebook: str = None
     dashboard: str = None
 
+    # if all are False then task is for all clouds
+    is_aws: bool = True
+    is_azure: bool = True
+    is_gcp: bool = True
```

Comment:
That's not a clean implementation. Capture the the cloud from the config class.

Reply:
We'll be getting a Cloud enum from SDK soon - see
- https://github.com/databricks/databricks-sdk-py/pull/390
- https://github.com/databricks/databricks-sdk-go/pull/700